src/utils/helpers/browser/browser.js

```diff
@@ -11,6 +10,13 @@ import Form from './../../../components/form';
 *
 */
 const Browser = {
+  isDomAvailable: () => {
+    return (!!(
+      typeof window !== 'undefined' &&
```

Comment:
Should use `Browser.getWindow()` or `global.window` at least.

Will need a test for this as well.